bucket/imagine.json

```
@@ -0,0 +1,49 @@
+{
+    "version": "1.1.4",
+    "description": "(1MB) Freeware Image & Animation Viewer for Windows",
+    "homepage": "https://www.nyam.pe.kr/dev/imagine/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.nyam.pe.kr/dev/imagine/"
+    },

```

```suggestion
    },
    "pre_install": "if (!(Test-Path \"$persist_dir\\Imagine.ini\")) { New-Item \"$dir\\Imagine.ini\" | Out-Null }",
```

`Imagine.ini` does not exist in the archive. Therefore we need to create it first.
Otherwise *Scoop* will create is **as a folder**. (by default)